src/PhpSpreadsheet/Style/NumberFormat.php

```diff
@@ -16,6 +16,7 @@ class NumberFormat extends Supervisor
     const FORMAT_NUMBER_00 = '0.00';
     const FORMAT_NUMBER_COMMA_SEPARATED1 = '#,##0.00';
     const FORMAT_NUMBER_COMMA_SEPARATED2 = '#,##0.00_-';
+    const FORMAT_NUMBER_COMMA_SEPARATED3 = '#,##0';
```

Comment:
Please use a more descriptive name for the constant, e.g. `FORMAT_NUMBER_COMMA_SEPARATED_INTEGER`.